auth.json.sample

```diff
@@ -1,8 +1,8 @@
 {
-    "http-basic": {
-        "repo.magento.com": {
-            "username": "<public-key>",
-            "password": "<private-key>"
-        }
+  "http-basic": {
```

Comment:
Seems like auth.json should also be 4 spaces as it generated using composer. Isn’t it?

Reply:
Seem you are right.
I have update my commit.